server/budgettracker.data/BudgetTrackerContext.cs

```diff
@@ -8,10 +8,11 @@ public class BudgetTrackerContext : DbContext
         public BudgetTrackerContext(DbContextOptions options)
             :base(options)
         {
-            
+
         }
 
         public DbSet<UserModel> Users { get; set; }
         public DbSet<BudgetModel> Budgets { get; set; }
+        public DbSet<BudgetDurationModel> BudgetDurations { get; set; }
```

Comment:
Is this the budget periods why do we need a separate table for BudgetDuration

Reply:
Lol maybe I should explain better what this PR is doing. I'll explain next time we get a chance in person.